protocol/x/prices/module.go

```diff
@@ -144,5 +149,15 @@ func (am AppModule) ExportGenesis(ctx sdk.Context, cdc codec.JSONCodec) json.Raw
 	return cdc.MustMarshalJSON(genState)
 }
 
+// PreBlock executes all ABCI PreBlock logic respective to the clob module.
+func (am AppModule) PreBlock(ctx context.Context) error {
+	defer telemetry.ModuleMeasureSince(am.Name(), time.Now(), metrics.PreBlocker)
+	PreBlocker(
+		lib.UnwrapSDKContext(ctx, types.ModuleName),
+		am.keeper,
+	)
+	return nil
+}
+
```

Comment:
sorry I think I messed up the function signature in my PR - it should be something like 

```
// PreBlock executes all ABCI PreBlock logic respective to the clob module.
func (am AppModule) PreBlock(ctx context.Context) (appmodule.ResponsePreBlock, error) {
	defer telemetry.ModuleMeasureSince(am.Name(), time.Now(), metrics.PreBlocker)
	PreBlocker(
		lib.UnwrapSDKContext(ctx, types.ModuleName),
		am.keeper,
	)
	return sdk.ResponsePreBlock{
		ConsensusParamsChanged: false,
	}, nil
}
```

you should see if you need `ConsensusParamsChanged` to be true

Reply:
```
// ResponsePreBlock represents the response from the PreBlock method.
// It can modify consensus parameters in storage and signal the caller through the return value.
// When it returns ConsensusParamsChanged=true, the caller must refresh the consensus parameter in the finalize context.
// The new context (ctx) must be passed to all the other lifecycle methods.
```

Reply:
I think `false` makes sense, because the cache isn't required for consensus is used for `ExtendVote`